src/routes/reset-password/+page.svelte

```diff
@@ -0,0 +1,137 @@
+<script lang="ts">
+    import * as Menu from 'menu/resetPassword';
+    import * as MenuItem from 'menu/menuItems';
+
+    import type { LoadResetPassword } from 'types/loadTypes';
+
+    import { apiUrl } from 'helper/links';
+
+    import Input from 'elements/input/input.svelte';
+    import Button from 'elements/input/button.svelte';
+    import PageWrapper from 'elements/section/pageWrapper.svelte';
+    import PasswordHints from 'elements/text/passwordHints.svelte';
+    import HeadlinePage from 'elements/text/headlinePage.svelte';
+    import Paragraph from 'elements/text/paragraph.svelte';
+    import StyledLink from 'elements/input/styledLink.svelte';
+    import Message from 'elements/text/message.svelte';
+    import { registerLookup } from 'lookup/registerLookup.js';
+
+    enum State {
+        Resettet,
```

Comment:
The past form of "reset" is also "reset". So you can either use `Reset` or `DidReset`.
```suggestion
        DidReset,
```